pcli/src/command/tx.rs

```diff
@@ -162,6 +165,24 @@ pub enum TxCmd {
     /// Currently, only zero-fee sweep transactions are implemented.
     #[clap(display_order = 990)]
     Sweep,
+
+    /// Perform an ICS-20 withdrawal.
+    #[clap(display_order = 1000)]
+    Ics20Withdrawal {
+        destination_chain_id: String,
+        destination_chain_address: String,
+        denom: String, //TODO: should we pull this out of amount
+        amount: String,
+        source_channel: String,
```

Comment:
Why do we need to specify the source channel explicitly? Could we infer it from the command?